lib/validation/rules/fileExtension.ts

```diff
@@ -1,11 +1,15 @@
 import { helpers } from '@vuelidate/validators'
+import { createRule } from '../Rule'
 import { fileExtension as baseFileExtension } from '../validators/fileExtension'
 
+export const message = {
+  en: 'The file extension is invalid.',
+  ja: 'ファイル拡張子が正しくありません。'
+}
+
 export function fileExtension(extensions: string[], msg?: string) {
-  return helpers.withMessage(
-    () => msg ?? 'The file extension is invalid.',
-    (value: File) => {
-      return !helpers.req(value) || baseFileExtension(value, extensions)
-    }
-  )
+  return createRule({
+    message: ({ lang }) => msg ?? message[lang],
+    validation: (value: File) => !helpers.req(value) || baseFileExtension(value, extensions)
```

Comment:
`!helpers.req(value)` may not be needed according to `createRule` implementation, but not so sure.

Reply:
Fixed: 5c20e70